tests/phpunit/MediaWiki/Content/SchemaContentTest.php

```diff
@@ -245,109 +245,4 @@ public function testPreSaveTransform() {
 		);
 	}
-
-	public function testFillParserOutput() {
-		$schema = $this->getMockBuilder( '\SMW\Schema\SchemaDefinition' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$schemaValidator = $this->getMockBuilder( '\SMW\Schema\SchemaValidator' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$schemaValidator->expects( $this->any() )
-			->method( 'validate' )
-			->will( $this->returnValue( [] ) );
-
-		$schemaFactory = $this->getMockBuilder( '\SMW\Schema\SchemaFactory' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$schemaFactory->expects( $this->any() )
-			->method( 'newSchema' )
-			->will( $this->returnValue( $schema ) );
-
-		$schemaFactory->expects( $this->any() )
-			->method( 'newSchemaValidator' )
-			->will( $this->returnValue( $schemaValidator ) );
-
-		$contentFormatter = $this->getMockBuilder( '\SMW\MediaWiki\Content\SchemaContentFormatter' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$title = $this->getMockBuilder( '\Title' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$title->expects( $this->any() )
-			->method( 'getDBKey' )
-			->will( $this->returnValue( 'Foo' ) );
-
-		$title->expects( $this->any() )
-			->method( 'getNamespace' )
-			->will( $this->returnValue( SMW_NS_SCHEMA ) );
-
-		$parserOptions = $this->getMockBuilder( '\ParserOptions' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$parserOutput = $this->getMockBuilder( '\ParserOutput' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$parserOutput->expects( $this->once() )
-			->method( 'setText' );
-
-		$parserOutput->expects( $this->once() )
-			->method( 'setIndicator' );
-
-		$instance = new SchemaContent(
-			json_encode( [ 'Foo' => 42 ] )
-		);
-
-		$instance->setServices( $schemaFactory, $contentFormatter );
-
-		$generateHtml = true;
-		$revId = 42;
-
-		$instance->fillParserOutput( $title, $revId, $parserOptions, $generateHtml, $parserOutput );
-	}
-
-	public function testFillParserOutput_MediaWikiTypeNotFoundException() {
-		$title = $this->getMockBuilder( '\Title' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$title->expects( $this->any() )
-			->method( 'getDBKey' )
-			->will( $this->returnValue( 'Foo' ) );
-
-		$title->expects( $this->any() )
-			->method( 'getNamespace' )
-			->will( $this->returnValue( SMW_NS_SCHEMA ) );
-
-		$parserOptions = $this->getMockBuilder( '\ParserOptions' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$parserOutput = $this->getMockBuilder( '\ParserOutput' )
-			->disableOriginalConstructor()
-			->getMock();
-
-		$parserOutput->expects( $this->once() )
-			->method( 'setText' );
-
-		$parserOutput->expects( $this->never() )
-			->method( 'setIndicator' );
-
-		$instance = new SchemaContent(
-			json_encode( [ 'Foo' => 42 ] )
-		);
-
-		$generateHtml = true;
-		$revId = 42;
-
-		$instance->fillParserOutput( $title, $revId, $parserOptions, $generateHtml, $parserOutput );
-	}
```

Comment:
Throwing out these poorly designed and brittle tests is reasonable enough. At the same time it's a pity to not have any good tests for the modified code.